Scripts/auth.py

```diff
@@ -44,16 +48,30 @@ def get_authenticated_service():
   API_VERSION = 'v3'
   DISCOVERY_SERVICE_URL = "https://youtube.googleapis.com/$discovery/rest?version=v3" # If don't specify discovery URL for build, works in python but fails when running as EXE
 
-  # Check if client_secrets.json file exists, if not give error
+  # Check if client_secrets.json file exists, if not give error and the option to add the file via a filepicker
   if not os.path.exists(CLIENT_SECRETS_FILE):
     print(f"\n         ----- {F.WHITE}{B.RED}[!] Error:{S.R} client_secrets.json file not found -----")
     print(f" ----- Did you create a {F.YELLOW}Google Cloud Platform Project{S.R} to access the API? ----- ")
     print(f"  > For instructions on how to get an API key, visit: {F.YELLOW}TJoe.io/api-setup{S.R}")
     print(f"\n  > (Non-shortened Link: https://github.com/ThioJoe/YT-Spammer-Purge/wiki/Instructions:-Obtaining-an-API-Key)")
-    input("\nPress Enter to Exit...")
-    sys.exit()
+    print(f"\n ")
+    print(f"\n > If a file has already been created then please input 1 to choose the file")
+    print(f"\n > Otherwise press Enter to exit")
+    inp = input("\nInput: ")
+    if inp == "1":
```

Comment:
And this to
`if inp.upper() in ("Y","YES")`

Reply:
Perhaps also check if input is "N"/"NO" before exiting - fat-fingering a button can happen 🤣

Reply:
if that happens then the program just exits because there are no other checks